CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 
 ## Release Highlights
 
+- Added support for semicolons in query strings.
```

Comment:
This is in the wrong place, you need to add a changelog entry to `changes since v7.4.0`

Reply:
Apologies.